### ISSUE TITLE
Request for Feedback: ``linkcheck`` module

### ISSUE THREAD

**Issue by jayaddison** [sphinx-doc/sphinx#11532]
From version 7.1.0, the [Sphinx `linkcheck` builder](https://www.sphinx-doc.org/en/master/usage/builders/index.html#sphinx.builders.linkcheck.CheckExternalLinksBuilder) has been updated to re-use HTTP connections, a change that should make linkchecking for Sphinx-based documentation projects more network-efficient, and possibly incrementally faster to complete.  See #11324 for a description of the feature.

We'd like to collect feedback about the effectiveness of the `linkcheck` builder, and especially to hear about any bugs as a result of these changes.  Please add a comment with your positive/negative feedback about the change in this issue thread, or file a separate issue if you have found a specific problem that we can address.

Thank you!
@jayaddison

**Comment by cvvergara** [comment]
In 7.1.1
I am getting on all anchors that:
```
(         contact: line    4) broken    https://trac.osgeo.org/osgeolive#Communication - Anchor 'Communication' not found
```
While the config file has, and they are not supposed to be checked
```
linkcheck_anchors = False
```

References:
The failing Job:
https://github.com/cvvergara/OSGeoLive-doc/actions/runs/5718357250/job/15493985854
The Sphinx version.
https://github.com/cvvergara/OSGeoLive-doc/actions/runs/5718357250/job/15493985854#step:5:329
The config file about anchors:
https://github.com/cvvergara/OSGeoLive-doc/blob/master/doc/conf.py#L224C1-L224C26
with sphinx 4.3.2 everything works fine
https://github.com/cvvergara/OSGeoLive-doc/actions/runs/5718544941/job/15494594335#step:5:288

**Comment by AA-Turner** [comment]
xref: https://github.com/sphinx-doc/sphinx/issues/11542

**Comment by jayaddison** [comment]
I think this can be closed - the report about a `linkcheck` regression in 7.1.0 was helpful and was resolved for the 7.1.2 release.
